src/model.ts

```diff
@@ -0,0 +1,468 @@
+import { VDomModel } from '@jupyterlab/apputils';
+import {
+  ColorPalette,
+  ColorInterpolationSpace,
+  ColorRGBA64,
+  contrastRatio,
+  hslToRGB,
+  rgbToHSL,
+  ColorHSL
+} from '@microsoft/fast-colors';
+
+const black = new ColorRGBA64(0, 0, 0, 1);
+const white = new ColorRGBA64(1, 1, 1, 1);
+const threshold = 7;
+
+function isBlack(color: ColorRGBA64) {
+  let test = 'Undefined';
+  if (color.r === 0.0 && color.g === 0.0 && color.b === 0.0) {
+    test = 'True';
+  } else {
+    test = 'False';
+  }
+  return test;
+}
```

Comment:
You can optimize the code by using boolean variable directly.

```suggestion
function isBlack(color: ColorRGBA64): boolean {
  return color.r === 0.0 && color.g === 0.0 && color.b === 0.0;
}
```